source/puppet/5.5/modules_documentation.markdown

```diff
@@ -194,116 +194,103 @@ To transition your module to the `REFERENCE.md` model, add Strings comments to y
 ```
-### `pet::cat`
+
+In the body of the `REFERENCE.md`, list out each of the module's elements (classes, defined types, functions, and so on), describe what it does, and list out its parameters.
+
+For each parameter, list:
+
+* If the parameter is required, specify *Required*.
+* The data type.
+* A description of what the parameter does, along with other information the user might need, such as "Only applies if `perl_enable => true`."
+* The valid values for the parameter, unless evident from the data type. For example, for a String data type, you should list specify what kind of string is needed (such as one of a specific set of strings or a valid URL). For a Boolean data type, you don't need to list true and false as valid values. 
+* The default value.
+ 
+``` markdown
+### Class: cat
+
+Installs and configures a cat in your environment.
 
 #### Parameters
 
+The following parameters are available in the `cat` class:
+
 ##### `purr`
 
-Data type: Boolean.
+Data type: `Boolean`.
 
 Enables purring in your cat.
 
 Default: `true`.
 
 ##### `meow`
 
-Enables vocalization in your cat. Valid options: 'string'.
-
-Default: 'medium-loud'.
+Data type: `Any`.
 
-#### `laser`
+Enables vocalization in your cat.
 
-Specifies the type, duration, and timing of your cat's laser show.
+Valid values: 'loud', 'medium-loud', 'kitten', 'silent'.
 
-Default: `undef`.
-
-Valid options: A hash with the following keys:
-
-* `pattern` - accepts 'random', 'line', or a string mapped to a custom laser_program, defaults to 'random'.
-* `duration` - accepts an integer in seconds, defaults to '5'.
-* `frequency` - accepts an integer, defaults to 1.
-* `start_time` - accepts an integer specifying the 24-hr formatted start time for the program.
+Default: 'medium-loud'.
 ```
 
+
 {:.concept}
-## Documentation style notes
+## Documentation style
 
 To write a clear, concise, and comprehensible documentation, we recommend following a few principles, best practices, and style guidelines.
 
-{:.section}
-### General principles of READMEs
-
-1. Write for both web and terminal viewing. We recommend [Markdown][commonmark]. Above all else, your module must be easily readable and scannable.
-2. Limit the number of external links. Linking to anything on the web limits the usability of the module, particularly if a range of users might use it in various environments, such as in terminal. Also, links look gross in plain Markdown and make your README less readable.
-3. Scannability is key. READMEs are formulaic and repetitious for a reason. Repetition means that no matter the module, users know where to get the information they're looking for.
+1. Write for both web and terminal viewing using [Markdown][commonmark]. Above all else, your module must be easily readable and scannable.
+1. Limit the number of external links. Linking to anything on the web limits the usability of the module, particularly when users read it in terminal.
```

Comment:
Something else I found along these same lines was that local links, or links to other docs in the module, will not work in the Forge display. The biggest example being linking from the Reference section in the README to the REFERENCE.md file. But also if you try to include links from REFERENCE.md to manifests or other files, they won't work on the Forge view. Dunno if we want to mention this.

Reply:
Should this be a Forge ticket to fix? Or is it just "won't work"?

Reply:
Filing FORGE-429 against this issue.